Doc/library/calendar.rst

```diff
@@ -586,12 +586,13 @@ The following options are accepted:
    or as an HTML document.
 
 
-.. option:: --first-weekday WEEKDAY, -f WEEKDAY
+.. option:: --first-weekday FIRST_WEEKDAY, -f FIRST_WEEKDAY
 
    The weekday to start each week.
    Must be a number between 0 (Monday) and 6 (Sunday).
    Defaults to 0.
 
+   .. versionadded:: 3.13
```

Comment:
@erlend-aasland We talked recently about the distinction between `versionadded` and `versionchanged`, which do you think we should have here?

https://devguide.python.org/documentation/markup/#paragraph-level-markup

Reply:
If it is applied to the option -- versionadded. If to the whole CLI -- versionchanged.

Reply:
Yes, this looks correct to me.